webpack.config.ts

```diff
@@ -612,6 +613,12 @@ if (
       },
     };
     appConfig.output!.publicPath = '/_static/dist/sentry/';
+
+    appConfig.plugins?.push(
+      new WebpackHookPlugin({
+        onBuildStart: ['node node_modules/@spotlightjs/sidecar/src/run.js'],
```

Comment:
would be nice if the package should export a bin script to use

Reply:
We actually do, fixed

Reply:
do we need to update here too @HazAT  ? https://spotlightjs.com/sidecar/webpack/

Reply:
good catch https://github.com/getsentry/spotlight/commit/cd64ee6c9cfa17291703e62f7f67e8ae405ecfd7